db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/BaseDataDictionary.java

```diff
@@ -185,18 +186,13 @@
     };
 
 	/**
-	 * List of procedures in SYSCS_UTIL schema with PUBLIC access
+	 * List of procedures and functions in SYSCS_UTIL schema with PUBLIC access
 	 */
-	protected static final String[] sysUtilProceduresWithPublicAccess = { 
+	protected static final String[] sysUtilRoutinesWithPublicAccess = { 
 												"SYSCS_INPLACE_COMPRESS_TABLE",
 												"SYSCS_COMPRESS_TABLE",
 												"SYSCS_MODIFY_PASSWORD",
-												};
-	
-	/**
-	 * List of functions in SYSCS_UTIL schema with PUBLIC access
-	 */
-	protected static final String[] sysUtilFunctionsWithPublicAccess = { 
+												"SYSCS_SAVE_SOURCECODE",
 												"SYSCS_PEEK_AT_SEQUENCE",
```

Comment:
I originally had a function to read from the syssourcecode table, but abandoned it as unnecessary.  Now this change is unnecessary too; pardon me.